src/librustc_unicode/char.rs

```diff
@@ -378,10 +500,32 @@ impl char {
     #[inline]
     pub fn is_xid_continue(self) -> bool { derived_property::XID_Continue(self) }
 
-    /// Indicates whether a character is in lowercase.
+    /// Returns true if this `char` is lowercase, and false otherwise.
     ///
-    /// This is defined according to the terms of the Unicode Derived Core
+    /// 'Lowercase' is defined according to the terms of the Unicode Derived Core
     /// Property `Lowercase`.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// let c = 'a';
+    /// assert!(c.is_lowercase());
+    ///
+    /// let c = 'δ';
+    /// assert!(c.is_lowercase());
+    ///
+    /// let c = 'A';
+    /// assert!(!c.is_lowercase());
+    ///
+    /// let c = 'Δ';
+    /// assert!(!c.is_lowercase());
+    ///
+    /// // The various Chinese languages do not have case, and so:
```

Comment:
Hmmm this is an interesting question, I guess it _is_ a property of a script and not a language